### ISSUE TITLE
charm duration bugged

### ISSUE THREAD

**Issue by thatnewguy3** [HomepointXI/Issues#375]
**Date & Time**: 4/24/19

**Client Version (use `/ver` in game):**  30190328_2

**Character Name:**

**Discord Name (if different from in game):**

**Nation:** 

**Job(level)/Sub Job(level):** BST occurred all levels

**NPC or Monster or item Name:** NA

**Zone name:**  all

**Coordinates (use `!where`):** all

**Item tool link (for items issues only):** NA

**Multi-boxing? (multiple clients on same connection):** no

**Steps to reproduce:**  Charm a monster and note how long it stays charmed. Even match, decent challenge, easy prey, and too weak monsters do not stay charmed for the duration they're supposed to.

**Any additional information:**

Charm duration appears to be bugged. When charming anything, duration does not match what it should be based on the ffxiclopedia wiki and my own experience. It's roughly 1/4 the duration it should be. Additionally, sometimes monsters de-spawn when they uncharm which is incorrect behavior.

**Comment by EpicTaru** [comment]
How much Charm + gear are you wearing when using the ability?

**Comment by EpicTaru** [comment]
How much CHR+ gear were you wearing when using the ability?

**Comment by thatnewguy3** [comment]
none. however, the base charm times even with no +charm gear are wrong.

see: https://ffxiclopedia.fandom.com/wiki/Charm

**Comment by thatnewguy3** [comment]
Not sure why this was tagged as a DSP issue. I didn't see it in any of the current issues on DSP's github and I haven't seen this issue on any of the other DSP servers I checked out before coming here.

**Comment by Jessland86** [comment]
It's a DSP issue because we have no custom code to do with charm. It's likely an issue with the revision the server is on, and might be fixed when we update next. 

Mobs despawning has to do with a mob being outside it's normal spawn area. This server has the setting enabled that depops mobs similar to retail vs allowing them to wander back to the spawn area. 

Depending on the other servers you have tried they might have custom work done to BST. I know Jova has a lot of stuff done custom to BST.

We'll see what charm looks like after we update and report to DSP if it's still broken.

**Comment by thatnewguy3** [comment]
fair enough. Although the mob despawning thing was just an aside to the main problem I noticed which is the very very short charm times. I suppose, as you say, seeing the state of charm after you guys update (when would that be out of curiosity?) can be a way to see if the issue gets resolved

**Comment by Heepster03** [comment]
![image](https://user-images.githubusercontent.com/8421725/131905280-28240fa0-6a4c-447d-bd75-de9c0e94cbdb.png)
Updated the core and tested on the TestServer. Charm is now working as intended and will be in the next update!

**Comment by EpicTaru** [comment]
This issue was resolved in yesterday's maintenance. Please open a new ticket should this issue re-occur.
